openapi/paths/evidenceFeed@user@{id}.yaml

```diff
@@ -27,6 +27,37 @@ get:
     default:
       $ref: ../components/responses/problem.yaml
   parameters:
+    - name: filter
+      in: query
+      schema:
+        description: |
+          Base 64 encoded object.
+          Decoded examples:
+           1. Return all feed items in category 'feedback'.
+              {"filterNames" => ["feedback"], "tag" => ‘"}
+           2. Return all feed items in category 'feedback' or 'achievements'.
+              {"filterNames" => ["feedback", "achievements"], "tag" => ""}
+           3. Return all feed items in category 'feedback' or 'learningMoments', that have tag "Class Work".
+              {"filterNames" => ["feedback", "learningMoments"], "tag" => "Class Work"}
+        example: eyJmaWx0ZXJOYW1lcyI6WyJsZWFybmluZ01vbWVudHMiLCJmZWVkYmFjayJdLCJ0YWciOiJDbGFzcyBXb3JrIn0
+        type: object
```

Comment:
Technically this isn't an object: it's a base64-decoded string which happens to decode to a JSON string. In order to reflect that more accurately:
- I'd set this to `type: string` and remove the `properties` completely
- I'd explain the meaning of `filterNames` and `tag` in `description`
- I'd document all `filterNames` possibilities in `description`

Reply:
There are other places like this and I've left them as objects in:
https://github.com/alaress/schoolbox-api-docs/pull/94
so that we can explain properties nicely.
I've seen these kind of examples somewhere...

Reply:
> There are other places like this

Whereabouts? That link points to this PR.

I'm a little worried about an end user of our documentation getting the wrong idea of how exactly this should be sent. I guess the `example` provided and the existing documentation works well enough though.

Reply:
If we really wanted to, we could follow the lead at https://swagger.io/docs/specification/describing-parameters/#schema-vs-content and do something like this, but I'm not sure it's correct either:

```yaml
parameters:
  - name: filter
    in: query
    content:
      application/json;base64:
        schema:
          ...
```

Reply:
Sorry, in this PR: https://github.com/alaress/schoolbox-api-docs/pull/92
- openapi/paths/assessment.yaml
- openapi/paths/evidenceFeed@user@{id}.yaml

Reply:
I guess the `filter` param _is_ an object, albeit an encoded one: I didn't fully understand the schema syntax.

Happy to leave as is for now, then: the description and example should be enough to communicate that it's base64-encoded.